antarest/study/web/raw_studies_blueprint.py

```diff
@@ -283,38 +283,35 @@ def aggregate_areas_raw_data(
         areas_ids: str = "",
         columns_names: str = "",
         current_user: JWTUser = Depends(auth.get_current_user),
-    ) -> t.Dict[str, t.Any]:
+    ) -> FileResponse:
```

Comment:
The frontend team prefers having cameCase names:

```python
output_id: str = Query(..., alias="outputId"),
query_file: AreasQueryFile = Query(..., alias="queryFile"),
frequency: MatrixFrequency = Query(...),
mc_years: str = Query("", alias="mcYears"),
areas_ids: str = Query("", alias="areasIds"),
columns_names: str = Query("", alias="columnsNames"),
max_aggregated_files: int = Query(10, alias="maxAggregatedFiles"),
```

> NOTE: the Ellipsis may be omitted, I think.

Reply:
Doing that, you have to change all the HTTP request parameters in the docstring and **unit test** 👀.

Reply:
This endpoint is not designed to be used by the front-end. I think we should let this the way it is and do what you recommend if the demand evolves in the future.